ui/core/sim.ts

```diff
@@ -256,10 +256,7 @@ export class Sim {
 				const newSpellEffects: SpellEffect[] = [];
 				const seenConsumableIds = new Set<number>();
 				const seenEffectIds = new Set<number>();
-				Object.entries(player.consumables ?? []).forEach(([field, cid]) => {
-					if (isDevMode()) {
-						console.log(field, cid);
-					}
+				Object.entries(player.consumables ?? []).forEach(([_, cid]) => {
```

Comment:
If you don't need the `field` anymore, then you can just use `Object.values()` instead.